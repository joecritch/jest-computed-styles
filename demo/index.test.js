const { parsePage } = require('../index');

test('it has the correct computed styles', () => {
  expect(parsePage('demo/test.html', 'demo/test.css')).toMatchSnapshot();
});
