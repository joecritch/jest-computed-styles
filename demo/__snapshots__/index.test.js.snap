// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has the correct computed styles 1`] = `
Array [
  Object {
    "computedStyle": Object {
      "display": "block",
      "margin": "0px",
      "padding": "0px",
    },
    "node": <body>
      <div
            class="hello"
      >
            Hello world
      </div>
</body>,
  },
  Object {
    "computedStyle": Object {
      "background": "blue",
      "display": "block",
      "margin": "0px",
      "padding": "0px",
    },
    "node": <div
      class="hello"
>
      Hello world
</div>,
  },
]
`;
